tests/darcy_velocity.prm

```diff
@@ -0,0 +1,157 @@
+#########################################################
+# This is a test for the new melt solver. It checks if
```

Comment:
Update the documentation here to reflect that the test is based on rising_melt_blob, but is modified to test the new Darcy flow implementation.  Likewise, update the documentation in other sections accordingly. Is there an analytical solution to this problem? If so, I think it would be good to add a benchmark example in this PR.